packages/block-editor/src/components/block-list/block-wrapper.js

```diff
@@ -62,26 +62,24 @@ const BlockComponent = forwardRef(
 		const { removeBlock, insertDefaultBlock } = useDispatch(
 			'core/block-editor'
 		);
-		const fallbackRef = useRef();
+		const wrapper = useRef();
 		const isAligned = wrapperProps && !! wrapperProps[ 'data-align' ];
-		wrapper = wrapper || fallbackRef;
 
-		// Provide the selected node, or the first and last nodes of a multi-
-		// selection, so it can be used to position the contextual block toolbar.
-		// We only provide what is necessary, and remove the nodes again when they
-		// are no longer selected.
+		// Provide the selected node so it can be used to position the contextual block toolbar.
+		// The wrapper changes when the alignment class changes.
 		useEffect( () => {
-			if ( isSelected || isFirstMultiSelected || isLastMultiSelected ) {
-				const node = wrapper.current;
+			const id = window.requestIdleCallback( () => {
```

Comment:
this function is not available on IE, we polyfill it somehow on @wordpress/priority-queue.

Reply:
Oh yeah, forgot about that